update_unix.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+// RunUpdate runs the update on the current tgf executable
+func RunUpdate() bool {
+	executablePath, err := os.Executable()
+	if err != nil {
+		printWarning("Error getting executable path", err)
+	}
+
+	currentDir := filepath.Dir(executablePath)
+
+	os.Setenv("TGF_PATH", currentDir)
+	os.Setenv("TGF_LOCAL_VERSION", version)
+
+	updateScript, fetchErr := fetchUpdateScript()
+	if fetchErr != nil {
+		printWarning("Error fetching update script: ", fetchErr)
+	}
+
+	output, errc := exec.Command("bash", "-c", updateScript).Output()
+
+	if errc != nil {
+		log.Fatal("Error running update script: ", errc.Error())
+		log.Fatal(string(output))
+	}
+
+	return strings.Contains(string(output), "Installing latest tgf")
```

Comment:
You are relying on output that may change here. You should rely on exit codes. If the update script doesn't return bad exit codes, you should modify it to do so

Reply:
I have tooled into this before, and doesn't seem to be support for it in go currently.. 
What I found was a bit hacky, I'll show you

Reply:
The script errors out when it fails. Your exitcode is in `errc`